metrics/value.go

```diff
@@ -53,9 +84,26 @@ func (v *gaugeVal) Type() string {
 	return "gauge"
 }
 
+func (v *gaugeVal) Keys(yield func(string) bool) {
+	for _, key := range []string{"value"} {
+		if !yield(key) {
+			return
+		}
+	}
+}
+
+func (v *gaugeVal) Value(key string) interface{} {
+	switch key {
+	case "value":
+		return v.Gauge.Value()
+	default:
+		return nil
+	}
```

Comment:
Minor, but style/structure differs from `*counterVal` implementation -- let's choose either "return nil outside of loop" or "return nil in default case" and use same approach for all implementations

Reply:
:+1: thanks, fixed